adapters/mocha.js

```diff
@@ -24,23 +24,29 @@ var attempted = 0;
 var finished = 0;
 var allPassed = true;
 
-module.exports = {
-  /**
-   * Setup server, then client.
-   */
+var adapter = module.exports = {
   before: function () {
     var rowdy = require("../index");
 
     // Setup server, then client.
     before(function (done) {
-      rowdy.setupServer(function (serverErr, server) {
-        if (serverErr) { return done(serverErr); }
+      // Check if actually using server.
+      if (!(rowdy.setting.server || {}).start) {
+        return done();
```

Comment:
You're the expert on that front. I would prefer seeing

```
done();
return;
```

Personally. Otherwise I was confused what `done()` was returning and how it would affect this function. Obviously it's just a `before()` function, but on first inspection I was confused what was going on. I suppose it doesn't matter either way, but that's what went through my mind. It could very well be a convention I just haven't seen before.